ansible/testbed_add_vm_topology.yml

```diff
@@ -103,7 +103,16 @@
     set_fact: hostname={{ VM_hosts | extract_hostname(topology['VMs'], VM_base, inventory_hostname) }}
     when: topology['VMs'] is defined
 
+  - name: Set properties list to default value, when properties are not defined
+    set_fact: properties_list=[]
+    when: configuration is not defined or configuration[hostname] is not defined or configuration[hostname]['properties'] is not defined
+
+  - name: Set properties list to values, when they're defined
+    set_fact: properties_list=configuration[hostname]['properties']
+    when: configuration and configuration[hostname] and configuration[hostname]['properties'] is defined
+
+  - debug: msg="{{ inventory_hostname }}"
```

Comment:
Do we need this debug here?
Probably it's better to use it as
debug: var=inventory_hostname
?